frontend/public/components/pod.tsx

```diff
@@ -181,58 +268,92 @@ const getHeader = (showNodes) => {
         sortField: 'metadata.name',
         transforms: [sortable],
         props: { className: tableColumnClasses[0] },
+        visible: true,
```

Comment:
We shouldn't need to define a `visible` prop here imo. We should calculate it from the column settings. If there are no column settings, you simply check `!additional`.

Reply:
@spadgett i think you're right...since this only drives the default column list for the cm dialog, any user modifications will be stored in state/local storage. At that point we will need to know visibility and additional, but not here.

Reply:
@spadgett I take that back, i had a problem defining default state in the initial pr. On this round of edits, Visible and additional are two properties that define the default state of the component and feeds into the col management modal for position in the two tables. I'm going to need them both after trying to get away from using visible.

Reply:
Can you expand on the problem? I think we can make it work, and it would be a cleaner API.